### ISSUE TITLE
Accidental deletion of model on the TX Ethos app.

### ISSUE THREAD

**Issue by itizmeunderscoreca** [FrSkyRC/ETHOS-Feedback-Community#4246]
I am at the field yesterday and I realize I don't have a idle up/down switch on the model I am going to fly. So I check how I have it configured on another plane with the same engine and then during my changing from the one model to the other I select set as current model and click that middle button, then I see the display delete model window appear and dissapear, model deleted, done, gone bye bye.

This morning I run a little software to undelete the file and I can't because the file has been partially overwritten by a log file from the following flight, another problem with the process so far as I am concerned.

How is it possible to delete a model with one click, no confirmation, no recycle bin nothing to undelete and then have your log files immediately write over the file on the next flight?

Of course it was a brand new setup, flown 1 memorial flight in May, had to complete the incomplete setup hence the activities of the day that were failed. Model was without a backup because there was no completed model to consider requiring a backup ( you know until yesterday if I was able to complete the setup )

I request, at the minimum, a two step process to delete a model. And perhaps have the deletions moved to a folder called something like deleted models where they could have a final deletion if desired.

X20Pro
TDSR18
103" Slick

**Comment by BladeScraper-Designs** [comment]
simply a "Trash" folder would be a good addition, IMO.  Anything deleted goes there... empty as desired, just like computers.

**Comment by miggy1952** [comment]
I have also had the same problem. I like the 'trash' suggestion

**Comment by bsongis-frsky** [comment]
There is already a confirmation dialog to confirm you really want to delete a model. I am still ok with the "Trash" folder idea

**Comment by itizmeunderscoreca** [comment]
I assure you I did not either select delete model nor did I click a second time to confirm deleting the model. I merely clicked when it said in highlighted text,  set current model.

I just went into themodel select menu and yes it does have a "are you sure you want to delete" function
I do not detect any screen delays between changes in the function to select like we see in many other screens
I have absolutely 0 ways to explain the deletion and am betting multiple users have had the same experience if I had it.

Note to self and others, make a clones tab and clone all models.

Edit: Sorry I forgot to say thank you for looking at this.

**Comment by spoke2570** [comment]
Ya. In the field it is super easy to do with noise, glare, wind in your face... I have totally deleted a model while trying to select a different one. 

Issue #3710 has further discussion.

**Comment by mawzthefinn** [comment]
It's not a permanent solution, but remember that long-pressing a model selects it, so you can use that instead of the tap & confirm to avoid accidental deletion or cloning

**Comment by frskyrc-ben** [comment]
> There is already a confirmation dialog to confirm you really want to delete a model. I am still ok with the "Trash" folder idea

Yes please have had several users request the 'Trash' folder so in case a model is still accidentally deleted after the confirmation dialog it can be recovered from Trash. Perhaps it can be like on the cell phone where the file is permanently deleted after 30-days
